CRM/Contact/BAO/GroupContact.php

```diff
@@ -340,7 +340,8 @@ public static function getContactGroup(
     $ignorePermission = FALSE,
     $onlyPublicGroups = FALSE,
     $excludeHidden = TRUE,
-    $groupId = NULL
+    $groupId = NULL,
+    $includeSmartGroups = FALSE
```

Comment:
@jainabhishek14 you need to also document the new parameter ```$includeSmartGroups``` in the function description as 
```
 @param bool $includeSmartGroups 
                     [short description]
```
Everything else is working fine :)